docs/project/project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.10")
```

Comment:
This is required by `docs/plugins.sbt` trying to load `./build.sbt:"sbt-plugin"` as a plugin in the docs project.